Tools/unicode/makeunicodedata.py

```diff
@@ -77,7 +77,7 @@
     "PDF", "EN", "ES", "ET", "AN", "CS", "NSM", "BN", "B", "S", "WS",
     "ON", "LRI", "RLI", "FSI", "PDI" ]
 
-EASTASIANWIDTH_NAMES = [ "F", "H", "W", "Na", "A", "N" ]
+EASTASIANWIDTH_NAMES = [ "N", "H", "W", "Na", "A", "F" ]
```

Comment:
Would it make sense to clarify it here as well (and also perhaps link the [specification](https://unicode.org/reports/tr11/#Unassigned))?